src/common/src/util/addr.rs

```diff
@@ -41,33 +40,33 @@ impl From<SocketAddr> for HostAddr {
 }
 
 impl TryFrom<&str> for HostAddr {
-    type Error = crate::error::RwError;
+    type Error = anyhow::Error;
```

Comment:
According to @BugenZhao, we have been doing this all the way along, but there is still `RwError` remaining in our code. Maybe we should tell everyone to stop using `RwError` and change it to other error types when encountering it. 🤣

Reply:
Nope, we're deprecating `RwError` but have no plan to replace it with `anyhow`. But it might be hard to determine an error type for this case. By convention, we use `String` or `anyhow` to assemble the bottom errors so LGTM here.